translations/sr_Cyrl/askvortsov-moderator-warnings.json

```diff
@@ -2,68 +2,68 @@
     "askvortsov-moderator-warnings": {
         "admin": {
             "permissions": {
-                "manage_warnings": "",
-                "view_warnings": "",
-                "delete_warnings": ""
+                "manage_warnings": "Направите и уклоните упозорења модератора",
+                "view_warnings": "Прикажи упозорења модератора",
+                "delete_warnings": "Трајно избришите упозорења модератора"
             }
         },
         "emails": {
             "post_warned": {
-                "subject": "",
-                "body": ""
+                "subject": "{warner_display_name} те је упозорио/ла са {стрикес, плурал, оне {# стрике} other {# упозорењем}} у дискусији {discussion_title}",
+                "body": "Hey {warnee_display_name}!\n\n{warner_display_name} те је упозорио/ла са {стрикес, плурал, оне {# упозорењем} other {# упозерењем}} у дискусији {discussion_title}\n\nDat је следећи razlog:\n\n---\n\n{public_comment}\n"
             },
             "user_warned": {
-                "subject": "",
-                "body": ""
+                "subject": "{warner_display_name} те је упозорио/ла {стрикес, плурал, оне {# упозорењем} other {# упозорења}}",
+                "body": "Hey {warnee_display_name}!\n\n{warner_display_name} те је упозорио/ла са {стрикес, плурал, оне {# упозорењем} other {# упозорења}}\n\n>Дат је следећи razlog:\n\n---\n\n{public_comment}\n"
             }
         },
         "forum": {
             "notifications": {
-                "warning_text": "",
-                "warning_no_strikes_text": ""
+                "warning_text": "{стрикес, плурал, оне {Упозорен/а си са # упозорења од стране {мод_усернаме}} other {Упозорен/а си са # упозорења од стране {мод_усернаме}}}",
+                "warning_no_strikes_text": "Упозорен/а си од стране {мод_усернаме} (но стрикес)"
```

Comment:
Placeholders (wrapped by `{}`) should not be translated. They occur independently and within the ICU syntax.